staging-process.md

```diff
@@ -0,0 +1,19 @@
+Anyone may submit an idea for a policy or program following the staging process outlined below. To submit an idea, create a pull-request and apply the appropriate staging label. 
+
+The proposal will be evaluated at bootstrap committee meetings and stage changes will occur at those times.
+
+
+## Stage 0 - Initial Idea
+Proposals in this stage are not necessarily written down - they may come from a conversation, an issue, or a brainstorming session. The purpose is to capture a thought that the community may want to explore further, and ideally to capture that idea for posterity.
+
+## Stage 1 - Proposal 
+Proposals in this stage are summarized in a markdown file. They have a short name and an identified a 'champion' - at least one person who will be pushing the proposal through to its conclusion. They have a description of the program idea or policy, what resources would be needed to execute it, what committee or role is responsible for overseeing it, ideas for metrics, execution, and so forth. They have clear language about which group or groups the program/policy will serve. The purpose of this stage is to put more color and shape to the Initial Idea, and begin capturing implementaiton or process questions that need to be answered.
+
+## Stage 2 - Draft 
+Proposals in this stage are written up in the style of a policy. They include examples of the program idea or policy in action, with clear steps and guidelines. They include the steps that will need to be taken to initially implement the idea or policy. The purpose of this stage is to refine the policy, make sure all stakeholders (including the board) have had a chance to review and request changes, and determine whether further iteration is needed. 
+
+## Stage 3 - Candidate 
+Proposals in this stage are near complete. They may be in a pilot or trial-run mode for a limited audience, or pending legal review. They may be awaiting budget approval. The purpose of this stage is to complete any operational checks, such as acquiring budget, access, or Board approval if needed, before launching the program.
+
+## Stage 4 - Complete
+The proposal is now an adopted policy or program of the Foundation, and is owned by the committee or role identified. It is expected that the policy or program will be re-evaluated on some regular basis and deprecated if need be.
```

Comment:
may I suggest a simpler process.

* Stage 0 - Initial proposal
* Stage 1 - The committee is reviewing the proposal
* Stage 2 - The proposal is being incorporated into the feedback provided by the committee

If there are competing Stage 1 proposals with no clear consensus, then both are combined into a single Stage 2 proposal that discusses both as options.

Reply:
That makes sense to me; does there need to be some process to figure out when it can go forward or what to do in a stalemate situation?

Related, should there be a timeline for proposals? Say something hasn't moved from Stage 0 in a year, it gets moved to an archive or folder or something?

Reply:
The stalemate situation is covered the last sentence of my comment there. Specifically:

> If there are competing Stage 1 proposals with no clear consensus, then both are combined into a single Stage 2 proposal that discusses both as options.